web/Makefile

```diff
@@ -9,20 +9,52 @@ NOTEBOOK_OUTPUT_DIR := docs/try-vast
 NOTEBOOK_INPUTS := $(wildcard $(NOTEBOOK_INPUT_DIR)/*.md)
 NOTEBOOK_OUTPUTS := $(addprefix $(NOTEBOOK_OUTPUT_DIR)/, $(notdir $(NOTEBOOK_INPUTS)))
 
-all: changelog notebooks
+DOT_GITHUB_OUTPUT_DIR := docs/develop-vast/contributing
 
-changelog: $(CHANGELOG_OUTPUT)
-
-notebooks: $(NOTEBOOK_OUTPUTS)
+all: changelog notebooks dot-github
 
-clean:
-	rm -f $(CHANGELOG_OUTPUT)
-	$(MAKE) -C $(NOTEBOOK_INPUT_DIR) clean
-	git clean -f -d -x $(NOTEBOOK_OUTPUT_DIR)
+changelog: $(CHANGELOG_OUTPUT)
 
 $(CHANGELOG_OUTPUT): $(CHANGELOG_INPUT)
 	cp -f $< $@
 
+notebooks: $(NOTEBOOK_OUTPUTS)
+
 $(NOTEBOOK_OUTPUTS): $(NOTEBOOK_INPUTS)
 	@$(MAKE) -C $(NOTEBOOK_INPUT_DIR) md
 	cp -r $(NOTEBOOK_INPUT_DIR)/output/md/* $(NOTEBOOK_OUTPUT_DIR)
+
+.github:
+	git clone https://github.com/tenzir/.github.git
+	cd .github && git pull
```

Comment:
The second command is redundant; right after the clone you already have the latest state, and later on the recipe will not be executed again because the file with that name already exists.

If you want do execute a pull on every make invocation, the way to do it would be something like this:

```
.github:
    git clone https://github.com/tenzir/.github.git

update-dot-github: .github
    cd .github && git pull

$(DOT_GITHUB_OUTPUT_DIR)/code-of-conduct.md: update-dot-github
	@echo "---\nsidebar_position: 0\n---\n" > $@
	@cat .github/$(@F) >> $@
```

Reply:
Good catch. I removed the redundancy.

I thought about doing something that pulls on every call initially, but then decided it's not worth the complexity. It's so rare that we change `.github` that it felt weird accounting for it in the common case.